packages/web-frontend/cypress/scripts/generateDashboardReportConfig.js

```diff
@@ -117,9 +124,7 @@ const getDashboardReportIdToGroups = async database => {
  * that matches the group's org unit code and level
  */
 export const generateDashboardReportConfig = async ({ database, logger }) => {
-  const reports = await database.executeSql(
-    `SELECT id, "viewJson"->>'name' as name from "dashboardReport"`,
-  );
+  const reports = await database.executeSql(`SELECT id, "drillDownLevel" from "dashboardReport"`);
```

Comment:
I guess `name` wasn't being used

Reply:
Indeed! In the latest version of this file I just `SELECT *` (I have pushed some commits after the initial PR)

Reply:
Ignore this - it gets changed to `*` in a later commit.